cli/core/test/e2e/framework/discovery_options.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package framework
+
+type DiscoveryOptions struct {
+	Name       string
+	SourceType string
+	URI        string
+}
+
+type PluginListInfo struct {
+	Description string `json:"description"`
+	Discovery   string `json:"discovery"`
+	Name        string `json:"name"`
+	Scope       string `json:"scope"`
+	Status      string `json:"status"`
+	Version     string `json:"version"`
+}
```

Comment:
Storing this struct under the `discovery_options.go` file doesn't make much sense. Let's create separate file for structs that are required for output processing.